lib/rules/jsx-no-target-blank.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(context) {
         var relFound = false;
         var attrs = node.parent.attributes;
         for (var idx in attrs) {
```

Comment:
@ljharb If I remember correctly `Object.keys` is only supported since Node.js 4.0